### PR TITLE
enable bookworm installation by adapting python version

### PR DESCRIPTION
**Pull request by Thovi98**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by Thovi98** [comment]
!testme

**Review by fflorent — APPROVED** [review]
Thanks!
Let's wait for the CI to complete and if it succeeds we'll merge

File: manifest.toml [hunk 1]
```diff
@@ -80,7 +80,13 @@ ram.runtime = "150M"
...
 
     [resources.apt]
-    packages = "python3.9, python3.9-venv"
+    packages_from_raw_bash = """
+    if [[ $YNH_DEBIAN_VERSION == "bullseye" ]]; then
+        echo "python3.9, python3.9-venv";
+    elif [[ $YNH_DEBIAN_VERSION == "bookworm" ]]; then
+        echo "python3.11, python3.11-venv";
+    fi
+    """
```
Comment (alexAubin) [inline manifest.toml:89]:
```suggestion
    package = "python3, python3-venv"
```

**Review by alexAubin — CHANGES_REQUESTED** [review]
This works but honestly this is just convoluted for nothing, let's just depend on `python3, python3-venv`, no need to specify the exact version...

File: manifest.toml (hunk 1 above)
Comment (Thovi98) [reply manifest.toml]: Agreed!

**Comment by fflorent** [comment]
!testme

**Review by fflorent — APPROVED** [review]

**Pull request merged**